examples/oneapi/cpp/source/knn/knn_cls_kd_tree_dense_batch.cpp

```diff
@@ -34,9 +34,7 @@ int main(int argc, char const *argv[]) {
   const auto y_train = dal::read<dal::table>(dal::csv::data_source{train_label_file_name});
 
   const auto knn_desc =
-      dal::knn::descriptor<float, oneapi::dal::knn::method::kd_tree>()
-          .set_class_count(5)
-          .set_neighbor_count(1);
+      dal::knn::descriptor<float, oneapi::dal::knn::method::kd_tree, oneapi::dal::knn::task::classification>(5, 1);
```

Comment:
may be better to save methods? I mean `set_class_count` and `set_class_count`? It looks more representative.